### DISCUSSION TITLE
Stongswan(Swanctl) in k8s is failing with error - connecting to 'unix:///var/run/charon.vici' failed: No such file or directory Error: connecting to 'default' URI failed: No such file or directory strongSwan 5.9.8 swanctl

### DISCUSSION THREAD

**Discussion by nishant2014** [strongswan/strongswan#1478] [Q&A]
Inside the kubernetes pod when I run swanctl -l command I get following error and looking for help

connecting to 'unix:///var/run/charon.vici' failed: No such file or directory
Error: connecting to 'default' URI failed: No such file or directory
strongSwan 5.9.8 swanctl

we have a config map with following contents -
apiVersion: v1
kind: ConfigMap
metadata:
  name: strongswan-conf
data:
  strongswan.conf: |
    charon {
            load_modular = yes
            plugins {
                    include strongswan.d/charon/*.conf
                    vici {
                     socket = unix:///var/run/charon.vici
                }
            }
           }

    include strongswan.d/*.conf

**Comment by Thermi** [comment]
We need the logs from inside the system to see any errors during startup. Also know that readiness check and so on needs to be delayed for some seconds until the daemon is actually started.
